sklearn/feature_selection/tests/test_mutual_info.py

```diff
@@ -7,12 +7,15 @@
 from sklearn.feature_selection._mutual_info import _compute_mi
 from sklearn.feature_selection import mutual_info_regression, mutual_info_classif
 
+DTYPES = (np.float64, np.float32)
 
-def test_compute_mi_dd():
+
+@pytest.mark.parametrize("dtype", DTYPES)
+def test_compute_mi_dd(dtype):
     # In discrete case computations are straightforward and can be done
     # by hand on given vectors.
-    x = np.array([0, 1, 1, 0, 0])
-    y = np.array([1, 0, 0, 0, 1])
+    x = np.array([0, 1, 1, 0, 0], dtype=dtype)
+    y = np.array([1, 0, 0, 0, 1], dtype=dtype)
```

Comment:
In the context of this test, x and y are discrete classification labels. The should be left as integer arrays.

Reply:
That's right. To be exact, both `x` and `y` are supposed to be discrete for this test.